src/GraphQL.Client.LocalExecution/GraphQL.Client.LocalExecution.csproj

```diff
@@ -1,17 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <Import Project="../src.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>A GraphQL Client which executes the queries directly on a provided GraphQL schema using graphql-dotnet</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL.MicrosoftDI" Version="4.6.1" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="4.6.0" />
-    <PackageReference Include="GraphQL.SystemReactive" Version="4.1.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="GraphQL.MicrosoftDI" Version="7.0.0" />
+    <PackageReference Include="System.Reactive" Version="5.0.0" />
```

Comment:
Do we want to retain the System.Reactive packageref?

Reply:
We actually don't use or build this library at all, so I guess it doesn't matter?  This library does use a lot of untranslatable System.Reactive functions, and is currently not compiling in develop

Reply:
Oh, we don't use it.  Neat.  For my education, what's the .LocalExecution library for?